app/views/admin/invoices/index.html.erb

```diff
@@ -1,5 +1,7 @@
-<h1>Invoice ID</h1>
+<h1>Invoice ID's</h1>
```

Comment:
Grammar thing (feel free to change later, it doesn't need to hold up this PR): 
If the header on this page is just referring to "all the ids" - you don't need the apostrophe, should be "Invoice IDs"

Reply:
Good point! Thanks for catching that.